src/test/java/org/opentripplanner/framework/lang/PredicateUtilsTest.java

```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.framework.lang;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+
+class PredicateUtilsTest {
```

Comment:
you could add a test that proves that duplicates are removed.

Reply:
8d39386af110705401a3d4764e072af34e47c945